test/core/metaflow_test/formatter.py

```diff
@@ -95,6 +95,10 @@ def _flow_lines(self):
             kwargs = ['%s=%s' % (k, v) for k, v in parameter.items()]
             yield 1, '%s = Parameter("%s", %s)' % (var, var, ','.join(kwargs))
 
+        for var, include in self.test.INCLUDE_FILES.items():
```

Comment:
I don't think this will work. Not every test has `INCLUDE_FILES`

Reply:
gah, forgot the defaults. Adding